CarAccident.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <stdexcept>
+
+using namespace std;
+
+class CarAccident {
+public:
+    static void main() {
+        try {
+            driveSafely();
+            cout << "Both persons reached the destination and are safe." << endl;
+        } catch (CarDamageException& e) {
+            handleCarDamage();
+        } catch (PersonInjuredException& e) {
+            handlePersonInjured();
+        } catch (BrakeFailureException& e) {
+            handleBrakeFailure();
+        } catch (...) {
+            cerr << "Unknown exception occurred." << endl;
+        } 
+    }
+
+private:
+    static void driveSafely() {
+        bool collision = false;
+        bool brakeFailure = true;
+
+        if (collision) {

```

Comment:
I am not able to figure out in which condition this code will process **PersonInjuredException** exception